ceph/ceph_admin/bootstrap.py

```diff
@@ -229,6 +229,8 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
             self.cluster.use_cdn = True
         elif custom_repo:
             self.set_tool_repo(repo=custom_repo)
+        elif build_type == "crimson":
```

Comment:
If we start using the upstream_repo setup, this would be redundant.

Reply:
All major changes in run.py have been removed